exporter/prometheusremotewriteexporter/helper.go

```diff
@@ -476,10 +476,25 @@ func addSingleDoubleSummaryDataPoint(pt *otlp.DoubleSummaryDataPoint, metric *ot
 	}
 }
 
+func orderBySampleTimestamp(tsArray []prompb.TimeSeries) []prompb.TimeSeries {
+	for i := range tsArray {
+		sL := tsArray[i].Samples
+		sort.Slice(sL, func(i, j int) bool {
+			return sL[i].Timestamp < sL[j].Timestamp
+		})
+		tsArray[i].Samples = sL
```

Comment:
Unnecessary, sort does sorting inplace so the slice does not change. (ptr, len, cap will be the same)

Reply:
Thank you. Indeed, unnecessary overkill after a prior change that I forgot to remove. Please take a look again.